loadtest.py

```diff
@@ -110,4 +110,5 @@ def test_crash_greater_than_20mb_uncompressed():
     resp = utils.post_crash(URL_SERVER, crash_payload)
     print("test_crash_greater_than_20mb_uncompressed: HTTP %s" % resp.status_code)
     # Verify HTTP 413
-    assert 413 == resp.status_code
+
+    assert 413 == resp.status_code, "Failure - HTTP %s, Page text: %s" % (resp.status_code, resp.text)
```

Comment:
Also, this specific test doesn't test Antenna. Anything over 20mb bounces off of nginx and won't affect how well Antenna performs under load.

I don't think we should be testing the > 20mb situation at all.